app/femr/data/daos/system/EncounterRepository.java

```diff
@@ -48,10 +49,11 @@ public IPatientEncounter createPatientEncounter(int patientID, DateTime date, in
 
 
         try{
-
+            // testing
```

Comment:
this comment doesn't provide info about what would need to change for not testing

Reply:
This was just here because I'm not sure if I should be creating this patientKey here or maybe instead I should pass one in from patient Item or something